src/types/types.d.ts

```diff
@@ -344,14 +344,60 @@ export enum Sep8ApprovalStatus {
   SUCCESS = "success",
 }
 
+export enum Sep8Step {
+  DISABLED = "disabled",
+  STARTING = "starting",
+  PENDING = "pending",
+  TRANSACTION_REVISED = "transaction_revised",
+  ACTION_REQUIRED = "action_required",
+  SENT_ACTION_REQUIRED_FIELDS = "sent_action_required_fields",
+  COMPLETE = "complete",
```

Comment:
💡 WDYT should we include `"no_further_action_required"` and `"follow_next_url "`?

Reply:
Not really, in this setup those are result types that can show up when we change from step `ACTION_REQUIRED` to `SENT_ACTION_REQUIRED_FIELDS`.